sdk/storage/azure-storage-blob-cryptography/src/test/java/com/azure/storage/blob/specialized/cryptography/EncryptedFlux.java

```diff
@@ -47,7 +48,7 @@
 
     public static final int DATA_COUNT = 40;
 
-    public static final int DOWNLOAD_SIZE = 64; // The total size of the "download" after expanding the requested range.
+    private static final int DOWNLOAD_SIZE = 64; // The total size of the "download" after expanding the requested range.
```

Comment:
Adding units in comments or in the constant name would make is easy to understand what 64 here represents. Bytes, KB  etc.
`DOWNLOAD_SIZE_IN_BYTES` or `DOWNLOAD_SIZE_IN_KB` etc.